src/html/components/concept.html

```diff
@@ -11,14 +11,14 @@ <h3>She gets to know you.</h3>
       <p>Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
       <p>This conversation unfolds over time, making this feel less like twenty questions and more like the start of a relationship. </p>
     </div>
-    <div class="messages">
+    <div class="message">
```

Comment:
This one would still be plural, because it's a list of `messages`, but each li below represents an individual `message`.